CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # 2020-01-08
 
+## API changes (relevant for self-hosters and client developers)
```

Comment:
How about splitting this section? As far as I can see, there is no change relevant for self-hosters, as no new configuration is introduced.

Reply:
I still keep 2 sections around but I find the release log confusing, happy to rethink it. Keeping a section "relevant for self hosters" only that depends on config changes sounds good.